core/src/main/java/com/homeaway/streamplatform/streamregistry/streams/GlobalKStreams.java

```diff
@@ -21,77 +21,72 @@
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
-import io.dropwizard.lifecycle.Managed;
-
 import org.apache.kafka.streams.KafkaStreams;
 import org.apache.kafka.streams.kstream.KStreamBuilder;
 import org.apache.kafka.streams.state.KeyValueIterator;
 import org.apache.kafka.streams.state.QueryableStoreTypes;
 import org.apache.kafka.streams.state.ReadOnlyKeyValueStore;
 
-import com.homeaway.digitalplatform.streamregistry.AvroStream;
 import com.homeaway.digitalplatform.streamregistry.AvroStreamKey;
-import com.homeaway.streamplatform.streamregistry.configuration.TopicsConfig;
 
 @Slf4j
-public class ManagedKStreams implements Managed {
+public class GlobalKStreams<T> {
```

Comment:
Should this be `T extends AvroStream`?

Reply:
Nope. I was making sure this is just doing a basic fetch of X from the store. We might have different stores with a different key later on, but the same logic overall.